src/contractTest/java/uk/gov/hmcts/reform/professionalapi/provider/OrganisationalInternalControllerProviderTest.java

```diff
@@ -207,4 +211,26 @@ private UserProfileCreationResponse getUserProfileCreationResponse() {
         userProfileCreationResponse.setIdamRegistrationResponse(201);
         return userProfileCreationResponse;
     }
+
+    @State("Organisations with payment accounts exist for given Pba status")
+    public void setUpOrganisationWithStatusForGivenPbaStatus() {
+        Organisation organisation = getOrganisationWithPbaStatus();
+        when(organisationRepository.findByPbaStatus(anyString())).thenReturn(Arrays.asList(organisation));
```

Comment:
@ishratt If the intention here is to return a list of organisations, then please use `List.of(organisation)`.